src/Generator/TracdelightCOM.php

```diff
@@ -285,117 +320,101 @@ private function returnImagePath($variation, int $number)
      */
     private function getVariationAttributes($variation, KeyValue $settings):array
```

Comment:
Isn't there a function in the EE helper to get the attributes?

Reply:
At the moment there's no function to get the specific AttributeValue of an attribute, depending on the market dependency. We planned with Max to change that maybe into an AttributeHelper

Reply:
ok!